app/daemon.go

```diff
@@ -198,10 +199,19 @@ func startManager(c *cli.Context) error {
 	router = handlers.ProxyHeaders(router)
 
 	listen := types.GetAPIServerAddressFromIP(currentIP)
-	logrus.Infof("Listening on %s", listen)
+	logger.Infof("Listening on %s", listen)
 
 	go http.ListenAndServe(listen, router)
 
+	go func() {
+		debugAddress := "127.0.0.1:6060"
```

Comment:
Should we allow customize debug server IP/port just in case?

Reply:
Agree in general that we should allow customization. With appropriate documentation:

For v1.1.2 I wanted to go with the simple approach, since we then don't need to document this just yet.
For v1.2 we should try to have a consistent design for our deployments and allow the user to enable the ability to expose the endpoint outside of the local pod host.

For example an ENV that by default is set to "127.0.0.1:6060" with explanations on how to make this 
- listen to all interfaces "0.0.0.0:6060" 
- listen to just the pod interface